Week2/index.js

```diff
@@ -0,0 +1,126 @@
+'use-strict';
+
+const prompt = require('prompt');
+const util = require('util');
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'world',
+});
+
+const execQuery = util.promisify(connection.query.bind(connection));
+const input = util.promisify(prompt.get.bind(this));
+
+async function queryDatabase() {
+  prompt.start();
+  try {
+    console.log(`
+    Please first enter the index number of the query, 
+    0. For capital city 
+    1. Region vs all spoken languages.
+    2. Language vs Number of cities.
+    3. Language vs Region.
+    4. Official Number of Languages vs Continent `);
+
+    const result = await input(['number']);
+    const input_number = result.number;
+
+    const select_query = [{
+        question: ` \nPlease enter the name of the country as input1 
+to find the capital city and leave input 2 blank \n`,
+        sql: `SELECT country.name, city.name
+        FROM country
+        LEFT JOIN city
+        ON country.capital = city.id
+        WHERE country.name = ?`,
+        prompts: ['country'],
+      },
+      {
+        question: `\nPlease enter the name of the region as input1
+to list all spoken languages and leave input 2 blank\n`,
+        sql: `SELECT countryLanguage.language
```

Comment:
The languages printed out are not unique, can you please address this?

Reply:
Ok, I will do so.